e2e/scenario_win_test.go

```diff
@@ -28,7 +36,10 @@ func Test_Windows2022Containerd(t *testing.T) {
 			VHD:             config.VHDWindows2022Containerd,
 			VMConfigMutator: func(vmss *armcompute.VirtualMachineScaleSet) {},
 			BootstrapConfigMutator: func(configuration *datamodel.NodeBootstrappingConfiguration) {
-
+			},
+			Validator: func(ctx context.Context, s *Scenario) {
+				ValidateFileHasContentWindows(ctx, s, "/k/kubeletstart.ps1", "--container-runtime=remote")
```

Comment:
it would be cool if each validator provided a windows and linux implementation which could be called here, something like: `ValidateFileHasContent.Powershell(...)` and `ValidateFileHasContent.Bash(...)` or similar

Reply:
oh, no please. I think 2 functions with a common prefix/suffix is much easier to follow than a chain of function calls like this.

Can be a single wrappper around checking if scenario is for a windows/linux. But I don't think it's necessary going to improve anything.

Reply:
I might change the validator to look at the OS running on the node and made decisions based on that. Because different versions of windows might have different powershell versions 😢

Reply:
Have combined them into a single wrapper.